testdata/data-files/server-side-eval/operators-date-equal-or-invalid.yml

```diff
@@ -0,0 +1,89 @@
+---
+name: operators - date - equal or invalid
+
+constants:
+  IS_MATCH:
+    value: true
+    variationIndex: 0
+    reason: { kind: "RULE_MATCH", ruleIndex: 0, ruleId: "ruleid" }
+  IS_NOT_MATCH:
+    value: false
+    variationIndex: 1
+    reason: { kind: "FALLTHROUGH" }
+  base_flag: &base_flag
+    on: true
+    fallthrough: { variation: 1 }
+    variations: [ true, false ]
+  base_rule: &base_rule
+    id: ruleid
+    variation: 0
+
+parameters:
+  # do permutations of each VALUE1 with each VALUE2; these are values that are either invalid as dates,
+  # or valid as dates but equal to each other
```

Comment:
This YAML is the equivalent of writing the following in JSON:
```json
[
  [
    {
      "VALUE1": "not valid"
    },
    {
      "VALUE1": true
    }, ...etc.
  ],
  [
    {
      "VALUE2": "not valid"
    }, ...etc.
  ]
]
```

The effect, as described [here](https://github.com/launchdarkly/sdk-test-harness/blob/master/docs/data_files.md), is to apply the whole file with {VALUE1: "not valid", VALUE2: "not valid"}, and then with {VALUE1: true, VALUE2: "not valid"}, etc., through every permutation.